parliament_challenge.py

```diff
@@ -0,0 +1,159 @@
+import requests
+import waitress
+import flask
+from flask import Flask, request, json
+
+
+def get_speeches_api():
+    """
+    Use speech's api to get speech resources.
+    Speech's api gets latest speeches and size determines how many resources to get.
+    Loop through dict to return all values in the "anforande" key.
+    These values include the keys and values of the speeches.
+    """
+
+    domain = 'http://data.riksdagen.se'
+    speeches = 'anforandelista'
+    size = 10
+    format_type = 'json'
+    response = requests.get('{}/{}/?anftyp=Nej&sz={}&utformat={}'.format(domain, speeches, size, format_type))
+    if response.status_code == 200:
+        data = response.json()
+        for k, v in data.items():
+            anforande = v["anforande"]
+            return anforande
```

Comment:
This doesn't make sense. It's a bug. Debug and see what is returned.!

Reply:
updated